src/cryptoadvance/specter/templates/includes/address-row.html

```diff
@@ -135,7 +141,13 @@
                 this.amount = parseInt(this.amount * 1e8);
             }
 
-            this.amountText.innerText = this.hideSensitiveInfo ? '########' : `${numberWithCommas(this.amount.toString())}`;
+            if (this.assetsCount > 1) {
+                this.amountText.innerText = this.hideSensitiveInfo ? '########' : `Multiple assets`;
+            } else if (this.assetsCount == 1) {
+                this.amountText.innerHTML = this.hideSensitiveInfo ? '########' : `${numberWithCommas(this.amount.toString())} <asset-label data-asset="${this.addressData.assets[0]}" data-label="${this.addressData.assetLabels[0]}"></asset-label>`;
```

Comment:
It is not required to pass asset label to the `<asset-label>` component, if it's not provided the component will fetch it automatically.
So passing `asset_labels` to all functions can be avoided - we can reduce amount of code.
```suggestion
                this.amountText.innerHTML = this.hideSensitiveInfo ? '########' : `${numberWithCommas(this.amount.toString())} <asset-label data-asset="${this.addressData.assets[0]}"></asset-label>`;
```